docs/installation.html

```diff
@@ -37,27 +49,27 @@ <h3>Prerequisites</h3>
 <p>Before installing, make sure you have the appropriate dependencies installed. First, there is OCaml:
 
 <ul>
-    <li>ocaml &ge; 4.08.0 &amp; &lt; 4.11.0</li>
+  <li>ocaml &ge; 4.08.0 &amp; &lt; 4.11.0</li>
 </ul>
 
 <p>The recommended way to install Ocaml is with OPAM. Instructions on downloading OPAM, and using OPAM to download Ocaml can be found <a href="https://ocaml.org/docs/install.html#OPAM">here</a>.
 
 <p>Next, there is the alpha release of BAP:
 
 <ul>
-    <li>bap 2.2.0-alpha</li>
+  <li>bap 2.2.0-alpha</li>
 </ul>
 
 <p>Instructions on downloading BAP from the testing repository can be found <a href="https://github.com/BinaryAnalysisPlatform/bap#installation">here</a>.
 
 <p>Then, there are the following OCaml packages:
 
 <ul>
-    <li>core_kernel 0.14.0</li>
-    <li>ounit2 2.2.3</li>
-    <li>z3 4.8.9</li>
-    <li>re 1.9.0</li>
-    <li>dune 2.7.1</li>
+  <li>core_kernel 0.14.0</li>
+  <li>ounit2 2.2.3</li>
+  <li>z3 4.8.9</li>
```

Comment:
We can remove `z3` from the list of things you need to install. It's installed by BAP now.